apix-integrationtests/src/main/java/eu/xenit/apix/rest/v2/tests/AllNodeInfoTest.java

```diff
@@ -39,15 +39,15 @@ public void setup() {
     public void testGetAllNodeInfo() throws IOException {
         HashMap<String, NodeRef> initializedNodeRefs = init();
         String url = makeNodesUrl(initializedNodeRefs.get(RestV2BaseTest.TESTFILE_NAME), "admin", "admin");
-        logger.info(" URL: " + url);
+        logger.debug(" URL: " + url);
         for (int i = 0; i < 20; i++) {
             logger.error("For the request of testGetAllNodeInfo");
             logger.error(url);
```

Comment:
Can't these be set to debug as well?

Reply:
hmm, might as well.
Will do.

Reply:
Tbh looking at it I think we could just delete that loop.